.github/workflows/ci.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        rust: ["1.42.0", "stable", "beta", "nightly"]
+        rust: ["stable"]
```

Comment:
This seems like a reasonable efficiency gain :ok_hand:

Reply:
thankfully github's parallism seems pretty great, but yeah, also means we are less stuck if rustfmt changes! 😆